### PR TITLE
Generate the SysML Data Model from SysML Gaphor Model

### PR DESCRIPTION
**Pull request by danyeaw**
This PR adds a profile_coder module that is used to generate a SysML data model in Python from the SysML gaphor model. The module uses loads the Gaphor model in to an ElementFactory and then generates the data model from there. This is part of completing #138.

This is a WIP, so far it just opens the Gaphor model and prints all of the Class definitions to a file, but doesn't include any attributes or inheritance.

### PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I have read, and am following the [Contributer guide](https://github.com/gaphor/gaphor/blob/master/CONTRIBUTING.md)
- [X] I have read and understand the [Code of Conduct](https://github.com/gaphor/gaphor/blob/master/CODE_OF_CONDUCT.md)

### PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes

### What is the current behavior?

Issue Number: N/A

### Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

File: gaphor/codegen/profile_coder.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    services = [
+        "element_dispatcher",
+        "event_manager",
+        "component_registry",
+        "element_factory",
+        "modeling_language",
+    ]
+    session = Session(services=services)
+    element_factory = session.get_service("element_factory")
+    modeling_language = session.get_service("modeling_language")
```
Comment (amolenaar) [inline gaphor/codegen/profile_coder.py:49]: I made a small fix on master, so we can just do:

```
element_factory = ElementFactory()
modeling_language = UMLModelingLanguage()
```

**Comment by amolenaar** [comment]
That's some really nice progress. Only need to resolve the Extensions, so they can be displayed as inheritance relationships.

We'll need a little function that orders the classes, so they are are written with parent classes at the top.

File: gaphor/codegen/profile_coder.py (hunk 1 above)
Comment (danyeaw) [reply gaphor/codegen/profile_coder.py:49]: Thanks for updating that! I'll merge master back in to this branch and start to make use of that.

**Comment by danyeaw** [comment]
My current ordering method was to just run through the list of classes three times:
1. Write UML imports
2. Write Classes with no inheritance
3. Write Classes with inheritance

I could be a little fancier and reorder the list of classes, I'll see if this makes the code more readable.

**Comment by amolenaar** [comment]
The model is generated. Looks like this can be merged, right?

**Pull request merged**